flink-runtime/src/main/java/org/apache/flink/runtime/heartbeat/DefaultHeartbeatMonitor.java

```diff
@@ -35,9 +36,9 @@
  *
  * @param <O> Type of the payload being sent to the associated heartbeat target
  */
-public class HeartbeatMonitorImpl<O> implements HeartbeatMonitor<O>, Runnable {
+public class DefaultHeartbeatMonitor<O> implements HeartbeatMonitor<O>, Runnable {
```

Comment:
> One small suggestion, can we rename HeartbeatMonitorImpl to DefaultHeartbeatMonitor to keep it consistent with other coordination interfaces?

Just my 2ct on that one: Do we have a common scheme for that one. Initially, I thought of commenting on `HeartbeatMonitorImpl` as well because I'm used to the `Default*` naming approach historically. But I noticed that even in `flink-runtime` we use `*Impl` quite often. I couldn't find anything in [Flink's coding conventions](https://flink.apache.org/contributing/code-style-and-quality-common.html). My impression is that other modules like `flink-streaming-java` make use of the `*Impl` scheme quite a bit.

```
$ cd ./flink
$ grep --include="*java" -Hrn "class [A-Za-z]*Impl " . | wc -l
154
$ grep --include="*java" -Hrn "class Default[A-Za-z]* " . | wc -l
176
```
It doesn't look like there's a clear winner. I'm just curious on your view on that one.

Reply:
To me, `*Impl` is **slightly** better, because `*Default` implies that there are non-default **production** implementations (which is not the case here). But here, consistency is more important.
Not sure if this (either `*Impl` or `*Default`) worth a naming convention and renaming the existing classes. WDYT?